lib/typescript/render/components/UnknownSourceText/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Linkify from 'linkifyjs/react';
+import styles from './index.module.scss';
+
+type TextRenderProps = {
+  text: string;
+  fromContact?: boolean;
+  sourceName: string;
+};
+
+export const UnknownSourceText = ({text, fromContact, sourceName}: TextRenderProps) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.unknownSourceHeader}>
+        <span className={styles.unknownSource}>{sourceName ?? 'Unknown'} Source</span>
+      </div>
+      <Linkify
```

Comment:
here we display the source name of the unknown source

Reply:
<img width="139" alt="Screenshot 2021-09-06 at 13 12 22" src="https://user-images.githubusercontent.com/38159391/132208836-e98a2d61-9a83-4b51-8342-faed31da0392.png">